tsinfer/inference.py

```diff
@@ -1296,14 +1305,16 @@ def convert_inference_mutations(self, tables):
                 metadata = _encode_raw_metadata(metadata)
             site_id = tables.sites.add_row(
                 site.position,
-                ancestral_state=site.alleles[0],
+                ancestral_state=site.ancestral_state,
                 metadata=metadata,
             )
             while mutation_id < num_mutations and mut_site[mutation_id] == site_id:
                 tables.mutations.add_row(
                     site_id,
                     node=node[mutation_id],
-                    derived_state=site.alleles[derived_state[mutation_id]],
+                    derived_state=site.alleles_ancestral_first[
```

Comment:
Yes, it would be clearer that we'd doing computation in the background there if we used a function on site for getting the ordered alleles.